### DISCUSSION TITLE
Blank screen after waking up from sleep

### DISCUSSION THREAD

**Discussion by mhserviscz** [SpiralLinux/SpiralLinux-project#129] [General]
Hi. Spiral Linux is an excellent way for me to use Debian. Unfortunately, just like on Debian or MX linux, (sometimes) I have the same problem with: 

1) a blank screen after waking up from sleep mode;only Ctrl + Alt + F1 (F2) will help,
2) the monitor does not turn off.

I tried using https://forums.debian.net/viewtopic.php?t=149965 but unfortunately it didn't help.

Could you help me pls?

_=======
System:    Host: XXX Kernel: 5.18.0-0.deb11.4-amd64 x86_64 bits: 64 
           Desktop: Xfce 4.16.0 Distro: Debian GNU/Linux 11 (bullseye) 
Machine:   Type: Laptop System: TOSHIBA product: TECRA Z50-A v: PT545E-00E011CZ 
           serial: <superuser required> 
           Mobo: TOSHIBA model: TECRA Z50-A v: Version A0 serial: <superuser required> 
           UEFI: TOSHIBA v: Version 4.30 date: 04/26/2018 
Battery:   ID-1: BAT1 charge: 42.1 Wh condition: 42.1/59.2 Wh (71%) 
CPU:       Info: Dual Core model: Intel Core i5-4300U bits: 64 type: MT MCP L2 cache: 3 MiB 
           Speed: 981 MHz min/max: 800/2900 MHz Core speeds (MHz): 1: 981 2: 2300 3: 1852 
           4: 1329 
Graphics:  Device-1: Intel Haswell-ULT Integrated Graphics driver: i915 v: kernel 
           Display: x11 server: X.Org 1.20.11 driver: loaded: modesetting unloaded: fbdev,vesa 
           resolution: 1920x1080 
           OpenGL: renderer: Mesa DRI Intel HD Graphics 4400 (HSW GT2) v: 4.5 Mesa 20.3.5 
Audio:     Device-1: Intel Haswell-ULT HD Audio driver: snd_hda_intel 
           Device-2: Intel 8 Series HD Audio driver: snd_hda_intel 
           Device-3: Microdia USB 2.0 Camera type: USB driver: snd-usb-audio,uvcvideo 
           Sound Server: ALSA v: k5.18.0-0.deb11.4-amd64 
Network:   Device-1: Intel Ethernet I218-LM driver: e1000e 
           IF: eth0 state: up speed: 1000 Mbps duplex: full mac: XXX 
           Device-2: Intel Wireless 7260 driver: iwlwifi 
           IF: wlan0 state: down mac: 06:2f:75:78:6d:a9 
Bluetooth: Device-1: Intel Bluetooth wireless interface type: USB driver: btusb 
           Report: ID: hci0 state: up running bt-v: 2.1 address: XXX 
Drives:    Local Storage: total: 523.56 GiB used: 51.46 GiB (9.8%) 
           ID-1: /dev/sda vendor: Western Digital model: WDS500G2B0A-00SM50 size: 465.76 GiB 
           ID-2: /dev/sdb type: USB vendor: Kingston model: DataTraveler 3.0 size: 57.8 GiB 
Partition: ID-1: / size: 465.46 GiB used: 43.06 GiB (9.3%) fs: btrfs dev: /dev/dm-0 
           ID-2: /boot/efi size: 299.4 MiB used: 3.2 MiB (1.1%) fs: vfat dev: /dev/sda1 
           ID-3: /home size: 465.46 GiB used: 43.06 GiB (9.3%) fs: btrfs dev: /dev/dm-0 
           ID-4: /tmp size: 465.46 GiB used: 43.06 GiB (9.3%) fs: btrfs dev: /dev/dm-0 
           ID-5: /var/log size: 465.46 GiB used: 43.06 GiB (9.3%) fs: btrfs dev: /dev/dm-0 
Swap:      ID-1: swap-1 type: zram size: 14.77 GiB used: 0 KiB (0.0%) dev: /dev/zram0 
Sensors:   System Temperatures: cpu: 50.0 C mobo: N/A 
           Fan Speeds (RPM): N/A 
Info:      Processes: 229 Uptime: 1d 52m Memory: 15.54 GiB used: 1.38 GiB (8.9%) Shell: Bash 
           inxi: 3.3.01 

/etc/default/grub
GRUB_DISABLE_OS_PROBER=false
GRUB_CMDLINE_LINUX_DEFAULT="init_on_alloc=0 quiet loglevel=0 cryptdevice=UUID=85569032-ccec-4daf-8056-5ce96c4b9ed5:luks-85569032-ccec-4daf-8056-5ce96c4b9ed5 root=/dev/mapper/luks-85569032-ccec-4daf-8056-5ce96c4b9ed5 splash"
GRUB_DISTRIBUTOR='Debian (SpiralLinux XFCE)'
GRUB_ENABLE_CRYPTODISK=y
=======_

Thank you...

**Comment by geckolinux** [comment]
Hi there, thanks for the report. Thanks for confirming that the same problem happens on vanilla Debian, which means that it's not an issue specific to SpiralLinux. There will always be some hardware that exhibits quirks with specific Linux distributions, and unfortunately it looks like yours is one of them. There probably is a workaround, specifically you might want to search for linux kernel boot parameters for suspend/sleep issues. But unfortunately it's not something that I could fix on SpiralLinux, as it will require trial and error on your specific hardware.
